deployments/tools/export/lakefs_export.py

```diff
@@ -78,11 +82,13 @@ def main():
     elif not has_branch and has_commit:
         source += args.commit_id + "/"
         reference = args.commit_id
-    else:
-        raise Exception("Should assign branch or commit, but not both.")
+    elif has_branch:            # and has_commit
+        error("Cannot set both branch and commit_id")
+    else:                       # not has_branch and not has_commit
+         error("Must set one of branch, commit_id")
```

Comment:
strange indentation here

Reply:
Thanks!  Turns out we never flake8 this.  Opening a _new_ bug to do that, but for now fixed _everything_ flake8 had to complain about.  So as punishment PTAL :-/